ccdImports/configFiles/CCD_Probate_Caveat/CaseEvent.json

```diff
@@ -9,6 +9,7 @@
   {"LiveFrom": "01/01/2017", "CaseTypeID": "Caveat", "ID": "createCasePaymentSuccess", "Name": "Create a case", "Description": "Create a case after payment success", "DisplayOrder": 8, "PreConditionState(s)": "CasePaymentFailed", "PostConditionState": "CaveatRaised", "SecurityClassification": "Public", "ShowSummary": "N"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "Caveat", "ID": "raiseCaveat", "Name": "Raise a caveat", "Description": "Raise a caveat against grant application", "DisplayOrder": 9, "PostConditionState": "CaveatRaised", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/caveat/raise", "RetriesTimeoutURLAboutToSubmitEvent": "30", "SecurityClassification": "Public", "ShowSummary": "Y", "EndButtonLabel": "Submit application"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "Caveat", "ID": "emailCaveatorForCaveatRaised", "Name": "Email caveator", "Description": "Email caveator", "DisplayOrder": 10, "PreConditionState(s)": "CaveatRaised", "PostConditionState": "CaveatRaised", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/caveat/general-message", "SecurityClassification": "Public", "ShowSummary": "N"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "Caveat", "ID": "withdrawCaveatForCaveatRaised", "Name": "Withdraw caveat", "Description": "Withdraw caveat", "DisplayOrder": 54, "PreConditionState(s)": "CaveatRaised", "PostConditionState": "CaveatClosed", "SecurityClassification": "Public", "ShowSummary": "N"},
```

Comment:
Check the display order here

Reply:
Yeah - I did not want to re-order everything... Ill change that. It does not matter - just makes merges more confusing. It still appears as the last event selection item

Reply:
Actually, we really dont need to do that, ill change the order number to be the last + 1